src/ray/util/url.cc

```diff
@@ -16,64 +16,103 @@
 
 #include <stdlib.h>
 
+#ifdef _WIN32
+#include <afunix.h>
+#else
+#include <sys/un.h>
+#endif
+
 #include <algorithm>
-#include <boost/asio/ip/address.hpp>
+#include <boost/asio/generic/stream_protocol.hpp>
+#ifndef _WIN32
+#include <boost/asio/local/stream_protocol.hpp>
+#endif
 #include <boost/asio/ip/tcp.hpp>
+#include <sstream>
```

Comment:
Same as above, the headers should be moved up (see https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes). I realize this was not the case before, but we should fix it.

Reply:
Done